migrations/20230613116515_courses.sql

```diff
@@ -0,0 +1,103 @@
+-- Courses Table
+CREATE TABLE IF NOT EXISTS public.courses (
+    id SERIAL PRIMARY KEY,
+    catalog_number SMALLINT NOT NULL,
+    subject_code VARCHAR(10) NOT NULL,
+    external_id VARCHAR(10) NOT NULL,
+    academic_level VARCHAR(13) NOT NULL,
+    title TEXT NOT NULL,
+    description TEXT NOT NULL,
+    abbreviated_description TEXT NOT NULL,
```

Comment:
I can't find any examples where the abbreviated description differs from the title. Would it be better to not store both then?